Makefile

```diff
@@ -109,8 +115,6 @@ ifeq ($(ARCH),amd64)
 	$(SED_I) "/CROSS_BUILD_/d" $(DOCKERFILE)
 else
 	# When cross-building, only the placeholder "CROSS_BUILD_" should be removed
-	# Register /usr/bin/qemu-ARCH-static as the handler for ARM binaries in the kernel
-	$(DOCKER) run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
See we don't need to do this in CI? I don't see any reference to `make register-qemu` in .travis.yml

Reply:
No, we don't use this in CI (it takes too long to run, that is why I build docker images using a VM)